samples/node/ConsoleApp/README.md

```diff
@@ -0,0 +1,21 @@
+# Node Console App
+This sample demonstrates a simple console app to be run in node. 
+
+Restore dependencies
+--------------------
+```
+npm install
+dotnet restore
+```
+Build the app
+-------------------
+This will start the Fable server and invoke webpack to build and bundle the app.
+```
+dotnet fable npm-run build
```

Comment:
Webpack will also do dead code elimination. 

I'm more wondering why someone would prefer webpack over rollup on a node only app in general? I.E. why not just use rollup as the default for node?